DomainContent/AvatarStore/wearableServer.js

```diff
@@ -32,7 +32,7 @@
                 userData: properties.userData,
                 parentID: entityID,
                 modelURL : properties.modelURL,
-                script: Script.resolvePath("attachmentItemScript.js"),
+                script: Script.resolvePath("attachmentItemScript.js") + "?" + Date.now(),
```

Comment:
Hmm, this would re-download the script for every attachment being created, are you sure you want to do this?

Reply:
I'll take it out before merging. Was trying to be crystal-clear on not caching for test